.github/workflows/release.yml

```diff
@@ -8,7 +8,7 @@ jobs:
   add-schemas-to-release:
     runs-on: ubuntu-22.04
     steps:
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v4
```

Comment:
Should we also update the `pull_request.yml` too?

Reply:
Yep, updated